### PR TITLE
Fixed script issues with Twinsen being able to interact with dead actors

### PR DESCRIPTION
**Pull request by xesf**
**Preview here:** https://pr-353.lba2remake.net

### CODE REVIEW

File: src/game/scripting/life.ts [hunk 1]
```diff
@@ -338,6 +342,7 @@ export function SUB_MAGIC_POINT(points) {
 }
 
 export function SET_LIFE_POINT_OBJ(actor, value) {
+    actor.props.runtimeFlags.isDead = false;
```
Comment (slwilliams) [inline src/game/scripting/life.ts:345]: What if value is 0?

File: src/game/scripting/life.ts [hunk 2]
```diff
@@ -583,6 +588,7 @@ export const SET_ARMOR_OBJ = unimplemented();
 export function ADD_LIFE_POINT_OBJ(index, points) {
     const actor = this.scene.actors[index];
     if (actor) {
+        actor.props.runtimeFlags.isDead = false;
```
Comment (slwilliams) [inline src/game/scripting/life.ts:591]: and I guess here too in the unlikely case that props.life is zero and points is zero we shouldn't set isDead to false.

**Review by slwilliams — APPROVED** [review]

File: src/game/scripting/life.ts (hunk 1 above)
Comment (xesf) [reply src/game/scripting/life.ts:345]: Yeah you are right, we should prevent it as well, even though they use a separate command to kill objects.

File: src/game/scripting/life.ts (hunk 2 above)
Comment (xesf) [reply src/game/scripting/life.ts:591]: Indeed, will do this both fixes.
Cheers

**Pull request merged**